py/janitor/schedule.py

```diff
@@ -348,7 +348,7 @@ async def do_schedule_regular(
 
     if normalized_codebase_value is None:
         normalized_codebase_value = await conn.fetchval(
-            "select coalesce(least(1.0 * value / (select max(value) from codebase), 1.0), 1.0) "
+            "select coalesce(least(1.0 * value / NULLIF((select max(value) from codebase), 0), 1.0), 1.0) "
```

Comment:
Why is this 0?

Reply:
NULLIF - compares 2 expressions and If they are equal, it returns NULL - otherwise return s the first expression.

So if `select max(value) from codebase` is 0, its then NULL (doesn't cause an issue) rather than 0 (which causes an issue).

Reply:
What I mean is, why are these values 0 to start with?